<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="chooseFile">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editSearch">
        <property name="text">
         <string>Search in format: &quot;Physics;Exam;1205&quot;, (e.g. &quot;Physics; ;1205&quot; is acceptable, place spaces instead of missing params)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonFind">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textOut"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Subject</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSubject"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboType">
        <item>
         <property name="text">
          <string>Lection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Practice</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lab</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Exam</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Classroom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineClassroom"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonFind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>findLesson()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>710</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addLesson()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>603</y>
    </hint>
    <hint type="destinationlabel">
     <x>722</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findLesson()</slot>
  <slot>addLesson()</slot>
 </slots>
</ui>
